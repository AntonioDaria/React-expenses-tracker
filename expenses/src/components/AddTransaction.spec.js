import React from 'react';
import { render } from '@testing-library/react';
import AddTransaction from './AddTransaction';



describe('TransactionList component', () => {
    let container;

    beforeEach(() => {
      ({ container} = renderAddTransaction());
    });

    it('should match the snapshot', () => {
        expect(container).toMatchSnapshot();
    });
});

const renderAddTransaction= () => {
    return render(
      <AddTransaction/>
    );
  };