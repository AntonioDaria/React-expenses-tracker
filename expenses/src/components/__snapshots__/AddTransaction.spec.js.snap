// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionList component should match the snapshot 1`] = `
<div>
  <h3>
    Add new transaction
  </h3>
  <form>
    <div
      class="form-control"
    >
      <label
        for="text"
      >
        Text
      </label>
      <input
        placeholder="Enter text..."
        type="text"
      />
    </div>
    <div
      class="form-control"
    >
      <label
        for="amount"
      >
        Amount 
        <br />
        (negative - expense, positive - income)
      </label>
      <input
        placeholder="Enter amount..."
        type="number"
      />
    </div>
    <button
      class="btn"
    >
      Add transaction
    </button>
  </form>
</div>
`;
